src/Pact/Eval.hs

```diff
@@ -612,8 +619,8 @@ installModule ModuleData{..} = do
         Interface{..} -> _interfaceName
   (evalRefs . rsLoadedModules) %= HM.insert n _mdModule
 
-msg :: Text -> Term n
-msg = toTerm
+msg :: Doc -> Term n
```

Comment:
Let's leave this alone (or delete it, conversely) -- it's not directly needed at usage sites to be part of this upgrade